rskj-core/src/test/java/org/ethereum/rpc/Web3ImplTest.java

```diff
@@ -300,27 +297,6 @@ public void getGasPrice()  {
         org.junit.Assert.assertEquals(expectedValue, web3.eth_gasPrice());
     }
 
-    @Test
-    public void sendRawTransaction() throws Exception {
```

Comment:
We can move the test, but we should keep testing what this test does

Reply:
sendRawTransaction was moved in another module https://github.com/rsksmart/rskj/pull/664/files#diff-1cda3e725d0aa668c33ffa5bf9232fc4R149 . Including new tests for that function .